lib/gooddata/lcm/actions/synchronize_user_filters.rb

```diff
@@ -89,32 +90,34 @@ def call(params)
             dry_run: false
           }
 
-          puts "Synchronizing in mode \"#{mode}\""
+          semaphore = Mutex.new
+
+          params.gdc_logger.info("Synchronizing in mode \"#{mode}\"")
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)